python/python-core/looping/using-else-in-loops.md

```diff
@@ -111,7 +111,7 @@ while x ??? 3:
 
 ## Revision
 
-An `else` statement is used in loops when the loops ??? is evaluated to ???.
+An `else` statement is used in loops when the loop's ??? is evaluated to ???.
```

Comment:
how do you feel about removing "loop's" entirely? it might make the question a bit harder though

```suggestion
An `else` statement is used in loops when the ??? is evaluated to ???.
```

Reply:
I feel the `loop's` word makes it more clear for beginners